test/functionalTest/cases/0947_v2_list_all_entities/GET_v2_entities.test

```diff
@@ -277,13 +277,16 @@ Date: REGEX(.*)
 03. GET /v2/entities
```

Comment:
This .test is currently using POST /v1/updateContext. Maybe we should change to use POST /v2/entities.